tests/yam/migration/restore_upgrade_env.pm

```diff
@@ -14,13 +14,14 @@ use migration 'reset_consoles_tty';
 
 sub run {
     # Restore the original value of the variables
-    foreach my $var (qw(AGAMA SCC_ADDONS SCC_URL VERSION)) {
+    my $env_content = '';
+    foreach my $var (qw(AGAMA SCC_ADDONS SCC_URL VERSION VIDEOMODE)) {
         if (get_var($var . "_ENV")) {
             set_var($var, get_var($var . "_ENV"));
-            record_info($var, $var . '=' . get_var($var));
+            $env_content .= "$var=" . get_var($var) . "\n";
         }
     }
-
+    record_info('ENV', $env_content) if $env_content;
```

Comment:
VERSION is mandatory so you don't need the `if`

Reply:
updated